manifests/mod/proxy_connect.pp

```diff
@@ -0,0 +1,8 @@
+class apache::mod::proxy_connect (
+  $apache_version  = $::apache::apache_version,
+) {
+  if versioncmp($apache_version, '2.4') >= 0 {
```

Comment:
i'm confused. this means we "succeed" to "load" this module?
shouldn't we fail here?